resources/public/api/conf/1.0/application.yaml

```diff
@@ -665,19 +688,140 @@ components:
           properties:
             type:
               $ref: '#/components/schemas/MessageType'
-  requestBodies:
-    MessageRequest:
-      description: Message that needs to be submitted for a specific excise movement
-      content:
-        application/vnd.hmrc.1.0+json:
-          schema:
-            $ref: '#/components/schemas/Message'
-    MessageWithTypeRequest:
-      description: Message with specific type that needs to be submitted for a specific excise movement
-      content:
-        application/vnd.hmrc.1.0+json:
-          schema:
-            $ref: '#/components/schemas/MessageWithType'
+    PreValidateTraderRequest:
+      required:
+        - exciseTraderValidationRequest
+      properties:
+        exciseTraderValidationRequest:
+          $ref: '#/components/schemas/ExciseTraderValidationRequest'
+    ExciseTraderValidationRequest:
+      required:
+        - exciseTraderRequest
+      type: object
+      properties:
+        exciseTraderRequest:
+          $ref: '#/components/schemas/ExciseTraderRequest'
+    ExciseTraderRequest:
+      required:
+        - exciseRegistrationNumber
+        - entityGroup
+        - validateProductAuthorisationRequest
+      type: object
+      properties:
+        exciseRegistrationNumber:
+          $ref: '#/components/schemas/ExciseReferenceNumber'
+        entityGroup:
+          $ref: '#/components/schemas/EntityGroup'
+        validateProductAuthorisationRequest:
+          $ref: '#/components/schemas/ValidateProductAuthorisationRequest'
+    EntityGroup:
+      type: string
+      enum: [ 'UK Record', 'EU Trader', 'EU Warehouse', 'EU Temporary Authorisation' ]
+      example: 'UK Record'
+    ValidateProductAuthorisationRequest:
+      type: object
+      required:
+        - product
+      properties:
+        product:
+          $ref: '#/components/schemas/ProductRequest'
+    ExciseTraderValidationSuccessResponse:
```

Comment:
Could we simplify the responses to just one response with optional fields and the relevant examples for success and failure just fill in the relevant fields?

Reply:
Tried this but the resulting schema looked too confusing - wasn't clear when each item was used. So left as is